client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -207,7 +207,7 @@ class WpcomChecklistComponent extends PureComponent {
 			completed: task.isCompleted,
 			siteSlug,
 			firstIncomplete,
-			buttonPrimary: firstIncomplete.id === task.id,
+			buttonPrimary: firstIncomplete && firstIncomplete.id === task.id,
```

Comment:
Perhaps I'm overthinking this, but I wonder if we'll ever have a situation like so:
```js
task = {
  id: undefined,
}

firstIncomplete = {
  id: undefined,
}

firstIncomplete && firstIncomplete.id === task.id // true
```

If that would be a potentiality we could do something like:
```js
const firstId = get( firstIncomplete, 'id' );
...
buttonPrimary: firstId && firstId === task.id,
```